backend/src/src-fossology/src/main/java/com/siemens/sw360/fossology/handler/FossologyFileHandler.java

```diff
@@ -180,6 +179,8 @@ public Release getStatusInFossology(String releaseId, User user, String clearing
                     updateInDb = false;
                 }
             }
+            Optional<FossologyStatus> maxStatus = nullToEmptyMap(release.getClearingTeamToFossologyStatus()).values().stream().max(FossologyStatus::compareTo);
```

Comment:
newlines would make this line of code more readeable, i.e.
```
            Optional<FossologyStatus> maxStatus = nullToEmptyMap(release.getClearingTeamToFossologyStatus())
                .values()
                .stream()
                .max(FossologyStatus::compareTo);
```

It might also be good to encapsulate this in a function.